setup.py

```diff
@@ -3,12 +3,12 @@
 from setuptools import find_packages
 
 setup(
-    name='dj-dna-streaming-python',
-    version='1.3.2',
+    name='djdp-streaming-python',
+    version='1.5.0',
     description='Dow Jones DNA Streaming Project',
     author='Chris Flesche',
     author_email='chris.flesche@dowjones.com',
-    url='https://github.com/dowjones/dj-dna-streams-python',
+    url='https://github.com/miballe/dp-streams-python',
```

Comment:
I think this is a change made for Miguel's fork that we don't want to merge back to this repo

Reply:
good catch. my bad, missed this one merging